website/pages/docs/security/security-models/core.mdx

```diff
@@ -0,0 +1,380 @@
+
+- **System Administrator** - This is someone who has access to the underlying infrastructure to the Consul cluster. 
+  Often they have access to SSH or RDP directly into a server within a cluster through a bastion host. Ultimately they
+  have read, write and execute permissions for the actual Consul binary. This binary is the same for server and client 
+  agents using different configuration files. These users potentially have sudo, administrative, or some other 
+  super-user access to the underlying compute resource. They have access to all persisted data on disk, or in memory. 
+  This would include ACL tokens, certificates, and other secrets stored on the system. Users like these are essentially 
+  totally trusted, as they have administrative rights to the underlying operating-system with the ability to configure, 
+  start, and stop the agent.
+
+- **Consul Administrator** - This is someone (probably the same System Administrator) who has access to define the 
+  Consul agent configurations for servers and clients, and/or have a Consul management ACL token. They also have total 
+  rights to all of the parts in the Consul system including the ability to manage all services within a cluster.
+
+- **Consul Operator** - This is someone who likely has restricted capabilities to use their namespace within a cluster.
+
+- **Developer** - This is someone who is responsible for creating, and possibly deploying applications connected, or 
+  configured with Consul. In some cases they may have no access, or limited capabilities to view Consul information, 
+  such as through metrics, or logs.
+
+- **User** - This is the end user, using applications backed by services managed by Consul. In some cases services may 
+  be public facing on the internet such as a web server, typically through a load-balancer, or ingress gateway. This is 
+  someone who should not have any network access to the Consul agent APIs.
+
+### Secure Configuration
+
+Consul's security model is applicable only if all parts of the system are running with a secure configuration; **Consul 
+is not secure-by-default.** Without the following mechanisms enabled in Consul's configuration, it may be possible to 
+abuse access to a cluster. Like all security considerations, administrators must determine what is appropriate for their
+environment and adapt these configurations accordingly.
+
+#### Requirements
+
+- **mTLS** - Mutual authentication of both the TLS server and client x509 certificates prevents internal abuse through 
+  unauthorized access to Consul agents within the cluster.
+
+  - [`verify_incoming`](/docs/agent/options#verify_incoming) - By default this is false, and should almost always be set
+    to true to require TLS verification for incoming client connections. This applies to both server RPC and to the 
+    HTTPS API.
+
+  - [`verify_incoming_https`](/docs/agent/options#verify_incoming_https) - By default this is false, and should be set 
```

Comment:
Is it worth mentioning that TLS for the API may be unnecessary if it is exclusively served over loopback connections?

Reply:
Ah i see this is called out below around line ~136.

Reply:
I think it's totally work re-stating that point, so I'll add it there too.